src/api/RestAPI.ts

```diff
@@ -244,6 +245,7 @@ export class RestAPI extends EventEmitter {
         const params: RequestInit = {
             headers: this.getHeaders(data, auth, payload, acceptType),
             method,
+            ...(auth?.useCredentials ? { credentials: "include" } : {}),
```

Comment:
made the `credentials`  request parameter optional so that it can be used only for the API that uses cookie authentication.